pkg/cloudevents/eventcontext_v1.go

```diff
@@ -15,28 +16,38 @@ const (
 	CloudEventsVersionV1 = "1.0"
 )
 
-// EventContextV1 represents the non-data attributes of a CloudEvents v0.3
+// EventContextV1 represents the non-data attributes of a CloudEvents v1.0
 // event.
 type EventContextV1 struct {
+	// ID of the event; must be non-empty and unique within the scope of the producer.
+	// +required
```

Comment:
I have ordered these in the order the spec talks about them now. The order changed.